src/client/containers/QuizResultPage/QuizResultPage.css

```diff
@@ -0,0 +1,45 @@
+.main-div {
```

Comment:
I think the name of this file needs changing to QuizResultPage.styles.css, as required in this guidelines page. https://github.com/HackYourFuture-CPH/Guide-IT#coding-guidelines-for-components